tests/expansion-panel/expansion-panel.html

```diff
@@ -16,7 +16,7 @@
                 <div class="k-expander-sub-title">South America</div>
                 <span class="k-expander-indicator k-icon k-i-arrow-chevron-down"></span>
             </div>
-            <div class="k-expander-content-wrapper k-display-none">
+            <div class="k-expander-content-wrapper k-d-none">
```

Comment:
You should not update markup in the /tests folder.  These are snapshots generated from the html components that are available in the `/packages/html` folder. In order to change the actual tests, please do so by modifying the real test in `/packages/html/src/expansion-panel/tests`.

Revert this change afterwards.